pkg/plugins/scorer/load_aware.go

```diff
@@ -13,58 +13,58 @@ import (
 )
 
 const (
-	// LoadAwareScorerType is the type of the LoadAwareScorer
-	LoadAwareScorerType = "load-aware-scorer"
+	// LoadAwareType is the type of the LoadAware
```

Comment:
```suggestion
	// LoadAwareType is the type of the LoadAware scorer
```
Might still be needed in the comment though not in the Go type name